src/test/java/kostka/moviecatalog/service/RatingServiceIntegrationTest.java

```diff
@@ -0,0 +1,70 @@
+package kostka.moviecatalog.service;
+
+import kostka.moviecatalog.MovieCatalogApplication;
+import kostka.moviecatalog.dto.MovieDto;
+import kostka.moviecatalog.dto.RatingDto;
+import kostka.moviecatalog.entity.Movie;
+import kostka.moviecatalog.exception.InvalidDtoException;
+import kostka.moviecatalog.exception.MovieNotFoundException;
+import kostka.moviecatalog.repository.MovieElasticSearchRepository;
+import org.assertj.core.api.Assertions;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.transaction.annotation.Transactional;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(classes = MovieCatalogApplication.class)
+@Transactional
+public class RatingServiceIntegrationTest {
+    private static final long NON_EXISTING_MOVIE_ID = 100L;
```

Comment:
Ale mezi náma, už jsem viděl i tohle failnout :D

Tak kdyby se ti někdy stalo, možná by to šlo nastavit na 0 nebo Int.max. Podle databáze asi.